src/kernel/thread/Scheduler.h

```diff
@@ -138,7 +140,7 @@ class Scheduler {
 
     PriorityPattern &priority;
 
-    Util::Array<Util::ArrayBlockingQueue<Thread *>> readyQueues;
```

Comment:
Please do not expose readyQueues as a public variable. If you need to access it, implement a getter-method.